Assets/PatchKit Patcher/Scripts/Utilities/LauncherUtilities.cs

```diff
@@ -0,0 +1,73 @@
+using System;
+using System.IO;
+using PatchKit.Unity.Patcher.Debug;
+using PatchKit.Unity.Utilities;
+
+namespace PatchKit.Unity.Utilities
+{
+    public class LauncherUtilities
+    {
+        private static readonly DebugLogger DebugLogger = new DebugLogger(typeof(LauncherUtilities));
+
+        public const string LauncherPathFileName = "launcher_path";
```

Comment:
Isn't there any other file to keep this kind of information?

Reply:
As far as I can tell, there doesn't seem to exist a "config" file which would contain the hardcoded constants.

This seems like a good place to keep this information - where it's relevant.

Reply:
1. Maybe it should be a json file instead in case we'd like to add more information to it?
2. If so, let's call it launcher_data.json maybe?